isp/bind/named.conf.local

```diff
@@ -0,0 +1,40 @@
+//
+// Do any local configuration here
+//
+
+// Consider adding the 1918 zones here, if they are not used in your
+// organization
+//include "/etc/bind/zones.rfc1918";
+
+view "internal" {
+  match-clients { 127.0.0.1; 172.23.44.0/24; };
+  recursion yes;
+  allow-query { 127.0.0.1; 172.23.44.0/22; };
+
+  zone "corp.mimosa.pi" IN {
+	type master;
+	file "/etc/bind/zones/db.corp.mimosa.pi";
+  };
+
+  zone "mimosa.pi" IN {
+  	type master;
+	file "/etc/bind/zones/db.mimosa.pi";
+  };
+
+  zone "pi" IN {
+  	type slave;
+	masters { 10.10.10.10; };
+  };
+
+  include "/etc/bind/named.conf.default-zones";
+};
+
+view "external" {
```

Comment:
Create external view. Only allow external users to query mimosa.pi and NOT internal domains/services.